src/celeritas/em/UrbanMscParams.cc

```diff
@@ -200,7 +200,7 @@ UrbanMscParams::calc_material_data(MaterialView const& material_view)
 
     UrbanMscMaterialData data;
 
-    double zeff = material_view.zeff();
+    double const zeff = material_view.zeff();
```

Comment:
Is there a reason to have all of these as `double` instead of `real_type`?

Reply:
Since it's all setup code on the CPU, and CPU floating point operations are no slower with double, we might as well compute with the extra precision before truncating? It probably doesn't matter in practice.